events/tests/test_event_get.py

```diff
@@ -1155,3 +1155,111 @@ def test_get_event_list_verify_registration_filter(
     event3.save()
     get_list_and_assert_events("registration=true", [event, event2])
     get_list_and_assert_events("registration=false", [event3])
+
+
+@pytest.mark.django_db
+def test_get_event_list_max_attendee_capacity_filter_gte(
+    api_client,
+    event,
+    event2,
+    event3,
+):
+    event.maximum_attendee_capacity = 10
+    event.save()
+    event2.maximum_attendee_capacity = 9
+    event2.save()
+    event3.maximum_attendee_capacity = 8
+    event3.save()
+
+    get_list_and_assert_events("max_attendee_capacity_gte=10", [event])
```

Comment:
This seems to test only the "equal" part of "greater than or equal", which means that this test should pass if, for whatever reason, I changed the `lookup_expr` parameter to `exact`. Applies to the other gte/lte tests as well.